### PR TITLE
Altered logging in koji_scraper

### PR DESCRIPTION
**Pull request by yashvardhannanavati**

### CODE REVIEW

File: scrapers/koji_scraper.py [hunk 1]
```diff
@@ -7,6 +7,7 @@
 import logging
 
 log = logging.getLogger(__name__)
+logging.basicConfig(level=logging.DEBUG)
```
Comment (mprahl) [inline scrapers/koji_scraper.py]: Can you put this in `scripts/scraper.py` instead? That way all scrapers that are run through `scraper.py` will log as debug.

File: scrapers/koji_scraper.py [hunk 2]
```diff
@@ -35,7 +36,6 @@ def get_koji_builds(self, start_date):
             WHERE events.time IS NOT NULL AND events.time >= '{}'
             """.format(start_date)
 
-        log.debug("Query:", sql_query.strip())
```
Comment (mprahl) [inline scrapers/koji_scraper.py]: +1 to removing this

File: scrapers/koji_scraper.py (hunk 1 above)
Comment (yashvardhannanavati) [reply scrapers/koji_scraper.py]: Sure, Done!

**Comment by mprahl** [comment]
:+1:

**Comment by yashvardhannanavati** [comment]
You are reviewing PR's on a Sunday night :P

**Pull request merged**